lib/core/src/server/utils/load-custom-babel-config.js

```diff
@@ -47,11 +47,15 @@ function loadFromPath(babelConfigPath) {
 }
 
 function isBabelLoader8() {
-  // eslint-disable-next-line import/no-dynamic-require,global-require
-  const babelLoaderPkg = require(resolveSync('babel-loader/package.json', {
-    basedir: process.cwd(),
-  }));
-  return satisfies(babelLoaderPkg.version, '>=8.0.0-0');
+  try {
+    // eslint-disable-next-line import/no-dynamic-require,global-require
+    const babelLoaderPkg = require(resolveSync('babel-loader/package.json', {
+      basedir: process.cwd(),
+    }));
+    return satisfies(babelLoaderPkg.version, '>=8.0.0-0');
+  } catch (e) {
+    return false;
```

Comment:
In what case there will be an error here ? do we want to hide it ?

Reply:
Good question @igor-dv. This happens when `babel-loader` isn't in the root of `node_modules`. When testing, this was sufficient to handle the error and still load successfully.